### DISCUSSION TITLE
Restore admin email and password

### DISCUSSION THREAD

**Discussion by TheMixerDK** [alexjustesen/speedtest-tracker#1077] [Q&A] [answered]
Hi

I just installed speedtest-tracker and tried to change the admin user so that the user got my email address and password, but something went wrong, so I can no longer log in, regardless of whether I use the default email and password, or I use what I entered.
Is it possible to remove everything from speedtest-tracker so that all traces of it are gone so I can reinstall it, or maybe just reseat the admin user info?
 
I have tried to remove speedtest-tracker by running the following commands:
docker-compose down
docker rmi -f mariadb:10
docker rmi -f ghcr.io/alexjustesen/speedtest-tracker:latest
docker system prune
 
However, I have not had much luck with resetting speedtest-tracker, so I can use the standard email and password to log in.

Don't know if it has anything to say, but I run docker on an Ubuntu server

Any help to resolve this issue would be greatly appreciated

TIA

**Comment by TheMixerDK** [comment, marked as answer]
Oh yes, I just had to make a post here, then I found the solution to the above :-D
The above problem is solved.

Thanks for making speedtest-tracker available to us mortals ;)
I will now start setting it up.

**Comment by CodeWithCJ** [comment]
how did you fix? I deleted the docker volume folder and re-installed. but still not working.
